arcgis-ios-sdk-samples/Scenes/Extrude graphics/ExtrudeGraphicsViewController.swift

```diff
@@ -74,9 +74,9 @@ class ExtrudeGraphicsViewController: UIViewController {
         let y = self.cameraStartingPoint.y + 0.25
         
         //creating a grid of polygon graphics
-        for i in 0...6 {
-            for j in 0...4 {
-                let polygon = self.polygonForStartingPoint(AGSPoint(x: x + Double(i) * (squareSize + spacing), y: y + Double(j) * (squareSize + spacing), spatialReference: nil))
+        for column in 0...6 {
```

Comment:
Since `column` and `row` are not actually used as integers, it would be nice to update the for loops to use `stride` instead:

```swift
for column in stride(from: 0.0, through: 6.0, by: 1.0) {
    for row in stride(from: 0.0, through: 4.0, by: 1.0) {
        let polygon = self.polygonForStartingPoint(AGSPoint(x: x + column * (squareSize + spacing), y: y + row * (squareSize + spacing), spatialReference: nil))
        self.addGraphicForPolygon(polygon)
    }
}
```